handwrite/svgtottf.py

```diff
@@ -166,14 +175,15 @@ def generate_font_file(self, filename, outdir, config_file):
         sys.stderr.write("\nGenerating %s...\n" % outfile)
         self.font.generate(outfile)
 
-    def convert_main(self, config_file, directory, outdir):
+    def convert_main(self, config_file, directory, outdir, metadata):
         try:
             self.font = fontforge.font()
         except:
             import fontforge
 
         with open(config_file) as f:
             self.config = json.load(f)
+        self.metadata = json.loads(metadata) or {}
```

Comment:
Do we need need json.loads here?

Reply:
The Metadata being passed to the command line is a json string. So yes, this is needed to convert it into a dict.

Reply:
Can't we do eval?

Reply:
We can, but it will cause an issue with boolean or none values:

```python
>>> json.loads(json.dumps({"hi": True}))
{'hi': True}
>>> eval(json.dumps({"hi": True}))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1, in <module>
NameError: name 'true' is not defined
```

json.loads is a better and safer way.